### PR TITLE
Updating NuGet Pkg Info

### PR DESCRIPTION
**Pull request by eddieSkywalker90**
- Updating info in preparation to upload NuGet Pkg to Nuget.org.

### CODE REVIEW

File: JankyLibrary/Janky.Configuration/Samples/ConfigurationSample.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Janky.Configuration.Samples
+{
+    // This will need to be a class implemented in your target project. Simply make a class called 'Configuration' and bring the code below into it.
+    public class ConfigurationSample
```
Comment (Chem-Star) [inline JankyLibrary/Janky.Configuration/Samples/ConfigurationSample.cs:7]: I am not sure if Janky.Configuration adds enough value that I would use it over standard configuration methods. 
I ended up stumbling upon this thread that makes things a bit easier. https://stackoverflow.com/questions/54767718/iconfiguration-does-not-contain-a-definition-for-getvalue

Janky:
Settings.GetSettings("SubsectionA", "SectionA");

vs 

Standard:
Root["SectionA:SubsectionA"];

Comment (eddieSkywalker90) [reply JankyLibrary/Janky.Configuration/Samples/ConfigurationSample.cs:7]: Agreed, the root way is definitely simpler. Removed/Deleted all references to Janky.Config project from JankyLibrary solution, including ReadMe.
- Also removed/replaced all references in the ShelfLife application, as posted in a similarly timestamped ShelfLife commit.

File: JankyLibrary/Janky.EmailManager/Program.cs [hunk 2]
```diff
@@ -1,12 +1,12 @@
-﻿
+﻿using System;
 
 namespace Janky.EmailManager
 {
     class Program
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/Program.cs:5]: Removed. Changed Application type in Project Options to "Library" instead "Console App" in order to be able to delete Program.cs (and Main method).

File: JankyLibrary/Janky.EmailManager/EmailManager.cs [hunk 3]
```diff
@@ -8,36 +8,31 @@
...
+                mailMessage.Body = new TextPart(email.ContentType)
                 {
-                    Text = message
+                    Text = email.Message
                 };
 
-                SendMessage(mailMessage, username, password);
+                SendMessage(mailMessage, email.Username, email.Password);
             }
             catch (Exception e)
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/EmailManager.cs:37]: Removed Catching on both methods,

File: JankyLibrary/Janky.EmailManager/EmailManager.cs [hunk 4]
```diff
@@ -46,18 +41,7 @@ public static void SendSupportEmail(string fromUserEmail, string toUserEmail, st
         }
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/EmailManager.cs:41]: Removed along with TryCatches in the comment below.

[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/EmailManager.cs:41]: Intention currently is for target-project to be source for all information.

File: JankyLibrary/Janky.EmailManager/EmailManager.cs [hunk 5]
```diff
@@ -8,36 +8,31 @@
...
         /// </summary>
-        /// <param name="password"></param>
-        /// <param name="subject"></param>
-        /// <param name="message"></param>
-        /// <param name="fromUserEmail"></param>
-        /// <param name="toUserEmail"></param>
-        /// <param name="username"></param>
-        public static void SendSupportEmail(string fromUserEmail, string toUserEmail, string username, string password, string subject, string message)
+        /// <param name="email"></param>
+        public static void SendSupportEmail(Email email)
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/EmailManager.cs:20]: Reduced to method name SendEmail. (Will be making commit to update ShelfLife repo to updated reference name).

File: JankyLibrary/Janky.EmailManager/Email.cs [hunk 6]
```diff
@@ -26,6 +27,9 @@ public Email(string fromEmail, string toEmail, string username, string password,
 
             if (message != "")
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/Email.cs:28]: Section of code has been removed/updated. Line has been adjusted per above comment of setting var value inside class creation. This is done for message + subject.

File: JankyLibrary/Janky.EmailManager/Email.cs [hunk 7]
```diff
@@ -13,8 +13,9 @@ public class Email
         public string Password;
         public string Subject = "No Subject";
         public string Message = "No Message";
+        public string ContentType = "plain";
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/Email.cs:17]: Great idea. Changes made.

File: JankyLibrary/Janky.EmailManager/Email.cs [hunk 8]
```diff
@@ -13,8 +13,9 @@ public class Email
...
 
-        public Email(string fromEmail, string toEmail, string username, string password, string subject = "", string message = "")
```
[the remark this answers is not in the document]
Comment (eddieSkywalker90) [reply JankyLibrary/Janky.EmailManager/Email.cs:17]: Good idea. 
Moved all 3 into EmailManager, and reworked Email/EmailManager. You now make an instance of EmailManager inside EmailHandler and pass in the above creds. After, you can then send email using EmailHandler.EmailManager.SendEmail(), still passing in any user-made Email object (which now only consists of ToEmail, Subject, and Message).

File: JankyLibrary/Janky.EmailManager/Email.cs [hunk 9]
```diff
@@ -1,9 +1,9 @@
...
     /// <summary>
-    /// Simple class to represent the data needed to construct an email.
-    /// These are typically generated in your target project in a email controller class (i.e. EmailHandler.cs).
-    /// These can be created, and then consumed by the JankyLibrary.EmailManager to send emails.
+    ///     Simple class to represent the data needed to construct an email.
+    ///     These are typically generated in your target project in a email controller class (i.e. EmailHandler.cs).
+    ///     These can be created, and then consumed by the JankyLibrary.EmailManager to send emails.
     /// </summary>
     public class Email
     {
```
[the remark this answers is not in the document]
Comment (Chem-Star) [reply JankyLibrary/Janky.EmailManager/Email.cs:9]: To be added in a future release.

**Review by Chem-Star — APPROVED** [review]
Tested in conjunction with Shelf Life. Looks good.

**Pull request merged**